image/tools/lib/component/resources.sh

```diff
@@ -39,10 +39,11 @@ function backup_resource {
             echo '---' > /tmp/${type}.yaml
             for obj in $(oc get ${type} -n ${ns} | tr -s ' ' | cut -d ' ' -f 1 |  tail -n +2); do
                 echo '-' >> /tmp/${type}.yaml
-                echo "$(oc get ${type}/${obj} -n ${ns} -o yaml --export | sed 's/^/  /')" >> /tmp/${type}.yaml
-                cat /tmp/${type}.yaml | gzip > ${dest}/archives/${ns}-${type}.yaml.gz
-                rm -f /tmp/${type}.yaml
+                echo "$(oc get ${type}/${obj} -n ${ns} -o yaml --export)" > /tmp/${ns}-${type}.${obj}.yaml
             done
+            (cd /tmp; find -name "*.yaml" | tar -cvzf  "${dest}/archives/${ns}-${type}.yaml.gz" -T -)
```

Comment:
This produces .tar.gz archive, and if one follows SOP and uses gunzip - it won't unpack this archive.

Reply:
ok but all other components are using tar we should update the sop then